pruner/example_programs/fabric_with_creduce/test.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+/home/roborobo/projects/gauntlet_repo/gauntlet/modules/toz3/pruner/tests/broken_p4c/24895c1b28a35352f1d9ad1f43878c4f7061d3ab -I /home/roborobo/projects/gauntlet_repo/gauntlet/modules/toz3/pruner/example_programs/fabric_with_creduce/ ./fabric.p4 2>&1 > /dev/null | grep "Compiler Bug"
```

Comment:
These paths are hardcoded. Please make them relative. A good way to do this is to always use some form of file path that is consistent. For example
https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itsel

Reply:
Alright. I was just testing creduce, and forgot to change it.

Reply:
Is this the file reduce uses to check whether the program is still producing a bug?

Reply:
Yes